pkg/cake/cake_test.go

```diff
@@ -0,0 +1,64 @@
+package cake
+
+import "testing"
+
+func TestBakeNoFlowerNoEggs(t *testing.T) {
```

Comment:
comments on tests?

Reply:
the variable names used here are ambiguous, what is t?